package.json

```diff
@@ -145,6 +145,7 @@
     "@vitejs/plugin-react-swc": "^3.3.0",
     "concurrently": "^8.0.1",
     "cypress": "^12.11.0",
+    "cypress-xpath": "^2.0.1",
```

Comment:
Just a warning that this package recently became deprecated, but it should be fine to keep using.

Reply:
We can remove this package when we will have data-test for the component involved in Cypress automation.